src/Scriban/Parsing/Lexer.cs

```diff
@@ -1620,6 +1658,149 @@ private void ReadVerbatimString()
             _token = new Token(TokenType.VerbatimString, start, end);
         }
 
+        private void ReadInterpolatedString()
+        {
+            var start = _position;
+            var end = _position;
+            char startChar = c;
+            bool readingInterpolation = !(c == '"' || c == '\'');
+            if (readingInterpolation)
+            {
+                startChar = _openingStringChars.Peek();
+            }
+            NextChar(); // Skip ", ', { or }
+            while (true)
+            {
+                if (c == '\\')
+                {
+                    end = _position;
+                    NextChar();
+                    // 0 ' " \ b f n r t v u0000-uFFFF x00-xFF
+                    switch (c)
+                    {
+                        case '\n':
+                            end = _position;
+                            NextChar();
+                            continue;
+                        case '\r':
+                            end = _position;
+                            NextChar();
+                            if (c == '\n')
+                            {
+                                end = _position;
+                                NextChar();
+                            }
+                            continue;
+                        case '0':
+                        case '\'':
+                        case '"':
+                        case '\\':
+                        case '{':
+                        case 'b':
+                        case 'f':
+                        case 'n':
+                        case 'r':
+                        case 't':
+                        case 'v':
```

Comment:
Add escape handling for `\{`

Reply:
I have actually added but I have forgotten to add several test before last commit and after amend, I have got a merge conflict. Being not experienced with Git, it apparently did not upload these changes after rebase.

Reply:
In other hand, I needed to handle it in regular strings, because interpolated strings without any interpolated expression are treated as regular strings.